homeassistant/components/yolink/strings.json

```diff
@@ -111,15 +111,15 @@
         },
         "tone": {
           "name": "Tone",
-          "description": "Tone before playing audio."
+          "description": "Tone before playing audio (if unset, no tone is played)."
         },
         "volume": {
           "name": "Volume",
-          "description": "Speaker volume during playback."
+          "description": "Speaker volume during playback (if unset, uses the current global volume)."
         },
         "repeat": {
           "name": "Repeat",
-          "description": "The amount of times the text will be repeated."
+          "description": "The amount of times the text will be repeated (if unset, will not repeat)."
```

Comment:
I think for the first and third we don't need to say what happens when unset, the second might need a bit of emphasis that this is only during playback. this way we don't have the awkward parenthesis

Reply:
Let me know if the changes make sense